Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39504&attempt_id=2e83a5d9-141b-4074-a182-53cc8bf600dc)

Attempted: soapysdr-with-plugins, soapysdr, soapyuhd, soapybladerf, soapyairspy, welle-io, soapyremote, soapyhackrf, limesuite

<details><summary>Partial log (click to expand)</summary><p>

```
                 from /build/source/build/welle-io_autogen/UVLADIE3JM/../../../src/CRadioController.h:39,
                 from /build/source/build/welle-io_autogen/UVLADIE3JM/moc_CRadioController.cpp:9,
                 from /build/source/build/welle-io_autogen/mocs_compilation.cpp:2:
/build/source/src/DabConstants.h:54:17: note: #pragma message: Git hash is not defined! Set it to "unknown"
 #pragma message "Git hash is not defined! Set it to \"unknown\""
                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: *** [CMakeFiles/Makefile2:100: CMakeFiles/welle-io.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
builder for '/nix/store/0qz0kdi8z772rp3ggkncx9wp5www3l2r-welle-io-1.0-rc2.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/0qz0kdi8z772rp3ggkncx9wp5www3l2r-welle-io-1.0-rc2.drv' failed
```
</p></details>

--- a/pkgs/applications/misc/welle-io/default.nix
+++ b/pkgs/applications/misc/welle-io/default.nix
@@ -42,7 +42,6 @@ in stdenv.mkDerivation {
     homepage = http://www.welle.io/;
     maintainers = with maintainers; [ ck3d ];
     license = licenses.gpl2;
-    platforms = with platforms; linux ++ darwin;
+    platforms = with platforms; [ "x86_64-linux" "i686-linux" ] ++ darwin;
   };
-
 }

